impl/java/json-jackson/src/main/kotlin/br/com/guiabolso/events/json/jackson/Jackson2JsonAdapter.kt

```diff
@@ -0,0 +1,92 @@
+package br.com.guiabolso.events.json.jackson
+
+import br.com.guiabolso.events.json.JsonAdapter
+import br.com.guiabolso.events.json.JsonDataException
+import br.com.guiabolso.events.json.JsonNode
+import br.com.guiabolso.events.json.JsonNull
+import br.com.guiabolso.events.model.Event
+import com.fasterxml.jackson.core.JsonProcessingException
+import com.fasterxml.jackson.core.exc.StreamReadException
+import com.fasterxml.jackson.databind.DatabindException
+import com.fasterxml.jackson.databind.json.JsonMapper
+import com.fasterxml.jackson.databind.type.TypeFactory
+import java.lang.reflect.Type
+
+class Jackson2JsonAdapter(configuration: JsonMapper.Builder.() -> Unit) : JsonAdapter {
+
+    private var mapper: JsonMapper
+
+    init {
+        mapper = JsonMapper.builder()
+            .addModule(GuiabolsoJsonNodeModule)
+            .addMixIn(Event::class.java, IgnoreEventGetters::class.java)
+            .apply(configuration)
```

Comment:
Reordenar as operações pra não permitir o usuário remover o Mixin:
```suggestion
            .apply(configuration)
            .addModule(GuiabolsoJsonNodeModule)
            .addMixIn(Event::class.java, IgnoreEventGetters::class.java)
```